build/prbuild.yml

```diff
@@ -111,7 +111,7 @@ jobs:
   - task: CmdLine@2
     displayName: 'Upload coverage to codecov.io'
     inputs:
-      script: $(USERPROFILE)\.nuget\packages\codecov\1.12.1\tools\win7-x86\codecov.exe -t $(CODECOV_TOKEN) -f **/*coverage.cobertura.xml
+      script: $(USERPROFILE)\.nuget\packages\codecov\$(CODECOV_VERSION)\tools\win7-x86\codecov.exe -t $(CODECOV_TOKEN) -f **/*coverage.cobertura.xml
```

Comment:
Would it make sense to have a script that auto-detects the version, either by inspecting the versions under .nuet\packages\codecov, or by examining the build file where it's specified? This approach feels like we need to manually tweak it every time we pick up a new drop :(

Reply:
Thanks for explaining!